lib/libraries/default/metric-combo.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-env browser */
+import c3 from 'c3';
+import { each } from '../../utils/each';
+import { prettyNumber } from '../../utils/pretty-number';
+
+export default class MetricCombo {
+  render() {
+    const {
+      container,
+      colors,
+      metricResults,
+      title,
+      tooltip,
+    } = this.config;
+
+    const columns = [];
+    columns[0] = [];
+    each(this.dataset.selectColumn(0), (c, i) => {
+      let cell;
+      if (i > 0) {
+        cell = new Date(c);
+      }
+      columns[0][i] = cell;
+    });
+    columns[0][0] = 'x';
+
+    each(this.data()[0], (c, i) => {
+      if (i > 0) {
+        columns.push(this.dataset.selectColumn(i));
+      }
+    });
+
+    const prevResult = (metricResults && metricResults.previous && metricResults.previous.result)
+      || 0;
+    const currResult = (metricResults && metricResults.current && metricResults.current.result)
+      || 0;
+    const count = currResult - prevResult;
+    const percentDifference = prevResult === 0 ? 'Not defined' : Math.round((count / prevResult) * 100);
```

Comment:
let's imagine scenario, that I'm counting singups.
Last week I had 0
This week I have 10
If I use this component - I'll see "Not defined' in the percentage change, even if both of my queries return correct results.
Let's use `-` instead of `not defined`